spec/unit/controllers/services/service_instances_controller_spec.rb

```diff
@@ -4001,6 +4023,12 @@ def verify_forbidden(user)
 
       describe 'permissions' do
         let(:user) { User.make }
+        let(:other_org) { Organization.make }
```

Comment:
Per the conversation that led to PR #1023, please prefer `target_org` over `other_org`; same for space.

Reply:
Thanks @elenasharma, it will be easier for us to fix this with a new commit on top rather than rebase everything, would you be open to merging this PR?